test/presubmit-tests.sh

```diff
@@ -29,10 +29,37 @@ export DISABLE_YAML_LINTING=1
 
 source $(git rev-parse --show-toplevel)/vendor/github.com/tektoncd/plumbing/scripts/presubmit-tests.sh
 
+function check_go_lint() {
+    header "Testing if golint has been done"
+
+    # deadline of 5m, and show all the issues
+    golangci-lint -j 1 --color=never run
+
+    if [[ $? != 0 ]]; then
+        results_banner "Go Lint" 1
+        exit 1
+    fi
+
+    results_banner "Go Lint" 0
+}
+
+function check_yaml_lint() {
+    header "Testing if yamllint has been done"
+
+    local YAML_FILES=$(find . -path ./vendor -prune -o -type f -regex ".*y[a]ml" -print)
+    yamllint -c .yamllint ${YAML_FILES}
```

Comment:
Nice! This just happened to be installed in our image already?

Reply:
Yes, yamllint binary was there already